src/pages/VeSDL/Votes.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  styled,
+  useTheme,
+} from "@mui/material"
+import { BigNumber } from "ethers"
+import { GaugeController } from "../../../types/ethers-contracts/GaugeController"
+import React from "react"
+import { VotesType } from "./OnChainVote"
+import { enqueuePromiseToast } from "../../components/Toastify"
+import { useTranslation } from "react-i18next"
+
+const tenDaysInSecond = 3600 * 24 * 10
+
+const VoteTableRow = styled(TableRow)(({ theme }) => ({
+  "td:first-of-type": {
+    borderTopLeftRadius: "10px",
+    borderBottomLeftRadius: "10px",
+  },
+  "td:last-child": {
+    borderTopRightRadius: "10px",
+    borderBottomRightRadius: "10px",
+    padding: 0,
+  },
+  backgroundColor: theme.palette.background.paper,
+}))
+
+interface VoteHistoryProps {
+  gaugeControllerContract: GaugeController | null
+  voteUsed?: BigNumber
+  votes: VotesType
+}
+
+export default function VoteHistory({
+  gaugeControllerContract,
+  votes,
+  voteUsed,
+}: VoteHistoryProps) {
+  const theme = useTheme()
+  const { t } = useTranslation()
+
+  const handleResetVote = async (gaugeAddress: string) => {
+    try {
+      if (gaugeControllerContract) {
+        const txn = await gaugeControllerContract.vote_for_gauge_weights(
+          gaugeAddress,
+          0, // to reset vote with zero
+        )
+        void enqueuePromiseToast(1, txn.wait(), "vote")
+      }
+    } catch (error) {
+      console.error("error on vote ==>", error)
```

Comment:
You should also notify the user via toast that we weren't able to complete the action

Reply:
✅